src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php

```diff
@@ -177,10 +168,10 @@ public function getResources(): array
     /**
      * Resets pagination on the current connection.
      */
-    private function resetPagination()
+    protected function resetPagination()
```

Comment:
why making it protected ?

Reply:
I was wondering if, instead of having all those phpversion check, it would be better to extend the Query class with clearer methods. I don't like very much all those `if (\PHP_VERSION_ID < 70300) {`